pytools/idfx_test.py

```diff
@@ -100,7 +100,9 @@ def __init__ (self):
 
     # transform all arguments from args into attributes of this instance
     self.__dict__.update(vars(args))
-    self.referenceDirectory = "reference"
+    self.referenceDirectory = os.path.join(idefix_dir_env,"reference")
+    # current directory relative to $IDEFIX_DIR/test (used to retrieve the path ot reference files)
+    self.thisDir=os.path.relpath(os.curdir,os.path.join(idefix_dir_env,"test"))
```

Comment:
"this dir" may be ambiguous depending on context
```suggestion
    self.testDir=os.path.relpath(os.curdir,os.path.join(idefix_dir_env,"test"))
```